.storybook/preview.ts

```diff
@@ -65,7 +69,7 @@ export const preview: Preview = {
         options: {
             storySort: {
                 method: 'alphabetical',
```

Comment:
I think that when we have the `storySort.order` it's not needed to have a `storySort.method` as well

Reply:
I'll check that, thanks 👍

Reply:
indeed, looks like it doesn't